scripts/setup.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# INSTRUCTIONS:
+# This script will move the pre-push hook from script folder to
+# the .git/hooks folder
+#
+# Run the script from the oppia-android root folder:
+#
+#   bash scripts/setup.sh
+#
+# NOTE: this script should be run once after the initial codebase setup
+
+# Move file from script folder to .git/hooks folder
+mv scripts/pre-push .git/hooks/
```

Comment:
Do we also need to make the hook executable?

Reply:
yes, the hook (pre-push file) is executable.

Reply:
Ah, but will Git retain the +x bit for files downloaded by other users? I think in Oppia web we use chmod +x to ensure the file is executable. Can you make sure we're following the same strategy for these files?

Reply:
No, I don't think we need chmod, I am copying the file in such a way where it becomes executable.

Reply:
Okay, let's check it in and see then. Thanks :)